### ISSUE TITLE
I can’t buy a board :(

### ISSUE THREAD

**Issue by fabien0102** [replayio/overboard#12]
I tried to buy a hoverboard but it failed with an error :(

**Comment by TejasQ** [comment]
Do you have a Replay for this?

**Comment by fabien0102** [comment]
Yes, here: https://app.replay.io/recording/overboard-d--7eab1306-2e20-43a7-a96e-107d2260ae24?point=18497557559522272877852027567212814&time=7967&focusRegion=eyJiZWdpbiI6eyJwb2ludCI6IjAiLCJ0aW1lIjowfSwiZW5kIjp7InBvaW50IjoiMzExNTM3ODExNTI3MzI4MzUxNTgyMTEwNTk1Mzg0NjA2NzIiLCJ0aW1lIjoxMzQxNn19
